subsys/bluetooth/controller/Kconfig

```diff
@@ -176,10 +176,10 @@ config BT_CTLR_SDC_RX_STACK_SIZE
 # CONFIG_BT_CTLR_DF is declared in Zephyr and also here for a second time,
 # to avoid BT_CTLR_DF_SUPPORT dependency.
 config BT_CTLR_DF
```

Comment:
@ryanjh : Any suggestion as to how we should make the new connection AoA transmitter feature experimental?

One way would be something like

`
config BT_CTLR_DF
  bool

config BT_CTLR_SDC_CONNECTION_AOA
  bool "blabla"
  select EXPERIMENTAL
  select BT_CTLR_DF
  select BT_CTLR_DF_CONN_CTE_TX

config BT_CTLR_SDC_CONNECIONLESS_AOA
  bool "blabla!
  select BT_CTLR_DF
  select BT_CTLR_DF_ADV_CTE_TX
`

Any opinion on this?

Reply:
agreed as not needed